include_core/AtomicSupport.hpp

```diff
@@ -14,6 +14,7 @@
  *
  * Contributors:
  *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ *    James Johnston (IBM Corp.)   - initial z/TPF Port Updates
```

Comment:
@mstoodle I think adding the contributor is ok but just checking with you as well.